developer-hub/src/reference_docs/extrinsic_encoding.rs

```diff
@@ -0,0 +1,156 @@
+//! # Constructing and Signing Extrinsics
+//!
+//! Substrate is configurable enough that extrinsics can take any format, in theory. In practice,
+//! runtimes tend to use our [`sp_runtime::generic::UncheckedExtrinsic`] type to represent extrinsics.
+//! In Polkadot, this is configured
+//! [here](https://github.com/polkadot-fellows/runtimes/blob/94b2798b69ba6779764e20a50f056e48db78ebef/relay/polkadot/src/lib.rs#L1478)
+//! at the time of writing.
+//!
+//! What follows is a description of how extrinsics based on this type are encoded into bytes. These
+//! bytes can then be submitted to a chain for inclusion in a block (this is how we make changes to
```

Comment:
You can link to the STF ref-doc and say that extrinsics are fed to the STF (as a part of the full block) and thus make changes to the state as a result.